src/components/ec-radio-btn/ec-radio-btn.story.ts

```diff
@@ -0,0 +1,104 @@
+import { ref } from 'vue';
+
+import EcRadioBtn from './ec-radio-btn.vue';
+import type { RadioBtnProps } from './types';
+
+export default {
+  title: 'Radio Button',
+  component: EcRadioBtn,
+};
+
+const basicArgs: Pick<RadioBtnProps, 'options'> = {
+  options: [
+    { value: 'y', label: 'Yes' },
+    { value: 'n', label: 'No' },
+  ],
+};
+
+export const basic = () => ({
+  components: { EcRadioBtn },
+  setup() {
+    const modelValue = ref<RadioBtnProps['modelValue']>('');
+    return {
+      args: basicArgs,
+      label: 'Select one options',
+      modelValue,
+    };
+  },
+  template: `
+    <div class="tw-px-24 tw-pt-4 tw-pb-24">
+      <ec-radio-btn v-bind="args" :label="label" v-model="modelValue"/>
+    </div>
+  `,
+});
+
+export const all = () => ({
+  components: { EcRadioBtn },
+  setup() {
+    const modelUnchecked = ref<RadioBtnProps['modelValue']>('');
+    const modelChecked = ref<RadioBtnProps['modelValue']>('y');
+    const modelDescription = ref<RadioBtnProps['modelValue']>('');
+    const modelCheckedDisabled = ref<RadioBtnProps['modelValue']>('y');
+    const modelUncheckedDisabled = ref<RadioBtnProps['modelValue']>('y');
+    const modelInlineRadioGroup = ref<RadioBtnProps['modelValue']>('y');
+    const modelInlineText = ref<RadioBtnProps['modelValue']>('y');
+    const modelError = ref<RadioBtnProps['modelValue']>('');
+    const optionsWithDescription = [
```

Comment:
```suggestion
    const optionsWithDescription: RadioBtnProps['options'] = [
```
And lets rename that RadioBtnProps to RadioButtonProps. With TS it is easy peasy.

Reply:
renamed types